internal/configuration/schema/access_control.go

```diff
@@ -33,16 +33,19 @@ func IsNetworkValid(network string) bool {
 
 // Validate validate an ACL Rule
 func (r *ACLRule) Validate(validator *StructValidator) {
-	if r.Domain == "" {
+
+	if len(r.Domains) == 0 {
 		validator.Push(fmt.Errorf("Domain must be provided"))
```

Comment:
```suggestion
validator.Push(fmt.Errorf("At least one domain must be provided"))
```

Reply:
I had thought about making this change with my original PR, although it felt weird to change this warning to talking about multiple when most users will likely have just one and when the docs primarily show a single value here. I feel like there is a slightly lowered chance of confusing someone by just telling someone the domain needed to be defined instead of how many, but I get it is probably 6 eggs, one half dozen type of thing.

But as the primary dev, just reply you'd like it this way and I'll make the change =)

Reply:
That makes sense. No big deal. I'm fine with the current version.